### PR TITLE
feat: Added Italian language

### PR DESCRIPTION
**Pull request by blackcat-917**
## Description:
Added Italian language

## Checklist:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you linted your code locally prior to submission?
* [x] Have you successfully ran app with your changes locally?

### CODE REVIEW

**Comment by blackcat-917** [comment]
@agalwood can you review (and merge) my pull request? (Can you also opt-in Hacktoberfest for this project?)

File: src/shared/locales/index.js [hunk 1]
```diff
@@ -5,87 +5,90 @@
  *
  * Please keep the locale key in alphabetical order.
  */
-export const availableLanguages = [
```
Comment (agalwood) [inline src/shared/locales/index.js:8]: Please keep the code indented to the original style.

**Review by agalwood — CHANGES_REQUESTED** [review]
Please adjust the code.

**Review by agalwood — APPROVED** [review]

**Comment by agalwood** [comment]
Thanks

**Pull request merged**